src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -194,7 +207,7 @@ private String getAnonEmail(FeedbackParticipantType type, String name) {
     }
 
     public String getAnonEmailFromEmail(String email) {
```

Comment:
While not introduced by you, rename this method and/or its parameter so that its clear that the email parameter should be a student's email (since instructors can be anonymous too)